x-pack/plugins/maps/server/register_integrations.ts

```diff
@@ -35,4 +35,45 @@ export function registerIntegrations(
     shipper: 'other',
     isBeta: false,
   });
+  customIntegrations.registerCustomIntegration({
+    id: 'ingest_geojson',
+    title: i18n.translate('xpack.maps.registerIntegrations.geojson.integrationTitle', {
+      defaultMessage: 'GeoJson',
+    }),
+    description: i18n.translate('xpack.maps.registerIntegrations.geojson.integrationDescription', {
+      defaultMessage: 'Upload GeoJson with Maps.',
+    }),
+    uiInternalPath: `${getFullPath('')}#?${OPEN_LAYER_WIZARD}=${WIZARD_ID.GEO_FILE}`,
+    icons: [
+      {
+        type: 'eui',
+        src: 'logoMaps',
+      },
+    ],
+    categories: ['upload_file', 'geo'],
+    shipper: 'other',
+    isBeta: false,
+  });
+  customIntegrations.registerCustomIntegration({
+    id: 'ingest_shape',
+    title: i18n.translate('xpack.maps.registerIntegrations.shapefile.integrationTitle', {
+      defaultMessage: 'Shapefile',
+    }),
+    description: i18n.translate(
+      'xpack.maps.registerIntegrations.shapefile.integrationDescription',
+      {
+        defaultMessage: 'Upload Shapefile with Maps.',
```

Comment:
> Upload a Shapefile with Elastic Maps.

Or, can you add a few words about what a Shapefile is? I'm not quite sure, but something along the lines of:

> Upload geospatial data in vector format with Elastic Maps.

Reply:
How about "Upload a Shapefiles with Elastic Maps.", again, I added a plural.

I don't think we need to add an explanation of shapefiles in the description. If you don't know what they are then you likely don't have any to upload anyways.

Reply:
And here:

> Upload Shapefiles with Elastic Maps.